docs/spec_draft.md

```diff
@@ -127,3 +127,52 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.constant
+
+`stablehlo.constant(value) -> result`
+
+### Semantics
+
+Produces a `result` tensor from a constant `value`.
+
+### Attributes
+
+| Name | Type |
+|-|-|
+| `value` | `tensor` of boolean, integer, floating-point or complex element types |
+
+### Operands 
+
+| Name | Type |
+|-|-|
+| `value` | Constant n-dimensional data of boolean, integer, floating-point, or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | `tensor` of boolean, integer, floating-point, or complex element types |
+
+### Constraints
+
+  * Supported shapes: all static shapes.
+
+### Examples
+
+```c
+%0 = stablehlo.constant dense<1> : tensor<pred>
+// %0: true 
+
+%1 = stablehlo.constant dense<0> : tensor<i32>
```

Comment:
(A) If we plan to keep the example snippet here, we can have better names `%result` etc., (B) Else we can replace it with links to the interpreter test file. 

Not relevant to the current PR: Both (A) or (B) can be handled in a serate PR.